courses/ARTS1422/meta.json

```
@@ -2,6 +2,20 @@
     "code": "ARTS1422",
     "name": "Data Visualization",
     "semesters": [
+        {
+            "season": "Spring",
+            "year": "2024",
+            "course_id": "ARTS1422.01",
+            "department": "信息科学与技术学院",
+            "credit": "3",
+            "hours": "48",
+            "prerequisite": "未知",
+            "evaluation": "Assignments: Coding and  Paper Presentation (20%)  • Quiz (10%)  • The quiz will be only with  a two-sided A4 cheat  sheet  • Top VIS Competition (10%)  • Course Project (60%)",
+            "teacher": [
+                "liquan"
+            ],
+            "ta": []
```

If there exists a TA, but you do not know who, please add "unknown". Leave this blank means there is no TA in this course.
```suggestion
            "ta": [
                "unknown"
            ]
```